conf/cassandra.yaml

```diff
@@ -1823,6 +1823,9 @@ unlogged_batch_across_partitions_warn_threshold: 10
 
 # Audit logging - Logs every incoming CQL command request, authentication to a node. See the docs
 # on audit_logging for full details about the various configuration options and production tips.
+# For BinAuditLogger and FileAuditLogger, the following optional parameters can be configured:
+# - key_value_separator: Specifies the key-value separator to use for the log message. (Defaults to ":")
```

Comment:
One thing that wasn't clear to me was where exactly these go, I tried:


```yaml
audit_logging_options:
    key_value_separator: "^"
    field_separator: "-"
```

and got :

```
ERROR [main] 2024-07-24 16:16:50,716 CassandraDaemon.java:910 - Exception encountered during startup: Invalid yaml. Please remove properties [field_separator, key_value_separator] from your cassandra.yaml
```

Instead can we change these comments to have them inline like:

```yaml
  logger:
    - class_name: BinAuditLogger
    # parameters:
    #  - key_value_separator: ...
```

Reply:
looks good now, thanks!